sqlair.go

```diff
@@ -100,8 +104,9 @@ type Iterator struct {
 	started bool
 }
 
-// Query takes a context, prepared SQLair Statement and the structs mentioned in the query arguments.
-// It returns a Query object for iterating over the results.
+// Query builds a new query from a context, a SQLair Statement and the input
+// arguments. The query is run on the database when one of Iter, Run, Get or
+// GetAll is executed on the Query.
```

Comment:
Question about go doc, is this going to show up as links in the documentation or do you have to use [sqlair.Iter], [sqlair.Run], etc.

Reply:
Oh nice, didn't know you could link like this, have added it all over the place.